src/parseTools.mjs

```diff
@@ -1093,7 +1117,11 @@ function ENVIRONMENT_IS_WORKER_THREAD() {
 
 addToCompileTimeContext({
   ATEXITS,
+  ATPRERUNS,
   ATINITS,
+  ATPOSTCTORS,
+  ATMAINS,
+  ATPOSTRUNS,
```

Comment:
Do do need to make these array available to the macro context as well as the function for adding to them?

Reply:
Yes, to make the compile time/runtime callback order consistent. I need the raw array so I can push the runtime callbacks to be the very first thing called. Alternatively, the `addAt<x>` function could take an optional argument that adds to the front of the queue instead of the back.